src/main/resources/db/migration/DEV/V1_0_0_202301100901__add_admin.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO users(username, password, firstName, lastName, role_id) VALUES ('mytesting','encrypted_password','My','Ngo','52b7c1c0-2432-414d-8839-28f58761567a')
```

Comment:
Cái thời gian của file migration này có vấn đề

Reply:
Removed file and add "insert" command into common